src/pages/WishList/WishListCard.js

```diff
@@ -0,0 +1,48 @@
+import { MdiClose, MdiStar } from "../../assets/Icon/icon";
+import "./WishList.css";
+
+export function WishListCard({wishlistDetail}) {
+
+    const {url, rating, name, description, originalPrice, discountedPrice, discount} = wishlistDetail
+
+    return (
+        <div className="product-card">
+            <div className="product-card-cont-a">
+                <img className="product-image" src={url} alt="avatar" />
+                <div className="product-rating-container">
+                <span className="product-rating">{rating}</span>
+                <MdiStar />
+                </div>
+                <MdiClose
+                className="wishlist-close"
+                onClick={() =>
+                    filterDispatch({
+                    type: "REMOVE_FROM_WISHLIST",
```

Comment:
Suggestion: we could store dispatch type inside a constant, this will help in avoiding typo errors and improve ease of usability.

`const removeFromWishlist = "REMOVE_FROM_WISHLIST"`

`filterDispatch({
     type: {removeFromWishlist}
      ....
})`